src/xenia/ui/window_gtk.cc

```diff
@@ -21,6 +21,8 @@
 #include "xenia/ui/virtual_key.h"
 #include "xenia/ui/window_gtk.h"
 
+#include "zlib/inftrees.h"
```

Comment:
Why is this needed?

Reply:
My editor imported unneeded stuff, already fixed :)